### PR TITLE
Remove long prose in yaml files

### PR DESCRIPTION
**Pull request by GayathriVenkatesh**
Solves issue #1315 

---

Reviewing this pull request? Check out our [Code Review Practices](https://github.com/ifmeorg/ifme/wiki/Code-Review-Practices) guide if you haven't already!

### CODE REVIEW

**Comment by julianguyen** [comment]
I saw that you're using `>-` for en.yml, let's use that for the other files too.

In addition, you're getting an error from `bundle exec bundle-audit check --ignore CVE-2015-9284 --update` in Circle CI because you need to rebase from master which has updates that address these vulnerabilities.

File: config/locales/it.yml [hunk 1]
```diff
@@ -343,129 +377,277 @@ it:
...
+      main_message_one: '%{app_name} è una community dedicata a esperienze
+      sulla salute mentale che incoraggia i propri utenti a condividere le loro
+      storie con alleati di cui ci si può fidare.'
+      message_text_one: 'Gli alleati sono le persone con cui interagiamo
+      quotidianamente includendo amici, familiari, colleghi, professori ed
+      esperti di salute mentale.'
+      main_message_two: 'Avere a che fare con la salute mentale è ciò che ci
+      rende umani. Ma per molti di noi, essere aperti al riguardo è spesso molto
+      difficile.'
+      message_text_two: '<p>Non tutti sono terapeuti. Le persone con cui abbiamo
```
Comment (julianguyen) [inline config/locales/it.yml]: Does this work? I would toggle the languages in your local dev environment to actually confirm. Please do this for the other languages!

I think you'll need to use `>-`.

**Comment by julianguyen** [comment]
Wait why did you close this?

**Comment by GayathriVenkatesh** [comment]
Oh sorry! Must have been a mistake.
Looks like a few irrelevant changes got pushed as well. Let me make a fresh PR with just the required commits!

**Pull request closed without merging**